### ISSUE TITLE
Library Update: TurboJPEG

### ISSUE THREAD

**Issue by butcherg** [butcherg/rawproc#5]
in place of jpeg6.

**Comment by barracuda156** [comment]
Has this been done?

**Comment by butcherg** [comment]
No, I'm still  using the original libjpeg interface in the code.  If you have a use case or lament, let me know and I'll look into it. 

My priority right now is looking into adding libheif...
